### DISCUSSION TITLE
fpPS4 with Assassins Creed Mirage v1 02 PS4-CUSA33151 download

### DISCUSSION THREAD

**Discussion by shrek88** [red-prig/fpPS4#145] [General]
So yea that download works on a homebrew ps4 so i hoped it would work but i do get some issues when starting.. anyone wanna shine some light?.. and yes i cut some of it out since its alot of text but these are the main issues? any fixes? xD
[Acmirage fpPS4 problem.txt](https://github.com/red-prig/fpPS4/files/13259044/Acmirage.fpPS4.problem.txt)
thats the full file attached

**[Warn]: file libSceXml.prx not loaded!
[Warn]: file libSceWebBrowserDialog.prx not loaded!
[Warn]: file libScePngDec.prx not loaded!
[Warn]: file libSceNpUtility.prx not loaded!

[Warn]: file libSceInvitationDialog.prx not loaded!
[Warn]: file libSceNpUtility.prx not loaded!
[Warn]: file libSceNpParty.prx not loaded!
[Warn]: file libSceJson2.prx not loaded!
[Warn]: file libSceJpegEnc.prx not loaded!
[Warn]: file libSceJpegDec.prx not loaded!
[Warn]: file libSceInvitationDialog.prx not loaded!
[Warn]: file libSceAudio3d.prx not loaded!**

**Comment by gandalfthewhite19890404** [comment]
Close, please
